pkg/tests/api/alerting/api_alertmanager_test.go

```diff
@@ -2067,6 +2078,23 @@ func TestIntegrationAlertmanagerStatus(t *testing.T) {
 	}
 }
 `
+	cfgWithoutAutogen := fmt.Sprintf(cfgTemplate, `{
```

Comment:
perhaps move it to another PR?

Reply:
The test will fail without this change, I could perhaps disable the simplfied routing FF here instead but fixing the test seems best.